gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -2442,4 +2456,15 @@ public Map<String, Long> getStatistics() {
     return statistics.entrySet().stream()
         .collect(toImmutableMap(Map.Entry::getKey, e -> e.getValue().get()));
   }
+
+  private static String getUgiUserName() {
+    try {
+      UserGroupInformation ugi = UserGroupInformation.getCurrentUser();
+      return ugi.getShortUserName();
+    } catch (IOException e) {
+      logger.atInfo().log(
+          "Ignoring IOException; failure to fetch current user info");
+      return null;
```

Comment:
Adding this as a warning, because if we throw error it might break workflows which we do not want as this is an extra check that we have added for only our use case and were not checking before.